packages/button/src/react/index.js

```diff
@@ -322,12 +322,17 @@ const whitelistProps = (props, whitelist) =>
 
 class Button extends React.Component {
   render() {
+    const whitelistedProps =
```

Comment:
I like that this is created as a local var.  Even better might be some "code as docs" with a function extraction called something like `preventOnClickForDisabledAnchorTag`.